lib/engine/game/base.rb

```diff
@@ -42,6 +42,8 @@ class Base
       GAME_RULES_URL = nil
       GAME_DESIGNER = nil
       GAME_PUBLISHER = nil
+      GAME_END_CHECK_STOCKMARKET = false
```

Comment:
Could we generalize this as a hash
{stockmarket: :full_round, bank: :full_round} and apply it to all games

Reply:
Good idea. I did that and also rewrote the code handling this a bit. I think it now will be easier to extend it with more variants.

Have also added light blue squares to the stock market (a la 1849) for game end triggering squares. It is now possible to have multiple such squares with different values.

Reply:
Hm... the light blue of 1849 was a different meaning....  So I might need another color for game end trigger in stock market.